server/src/main/java/org/elasticsearch/index/mapper/MapperFeatures.java

```diff
@@ -25,6 +25,7 @@ public class MapperFeatures implements FeatureSpecification {
     // Used to avoid noise in mixed cluster and rest compatibility tests. Must not be backported to 8.x branch.
     // This label gets added to tests with such failures before merging with main, then removed when backported to 8.x.
     public static final NodeFeature BWC_WORKAROUND_9_0 = new NodeFeature("mapper.bwc_workaround_9_0");
+    public static final NodeFeature SPARSE_VECTOR_STORE_SUPPORT = new NodeFeature("mapper.sparse_vector.store_support");
```

Comment:
Nit: The general pattern for features is to define them in the class the implements the new functionality (in this case, `SparseVectorFieldMapper`)

Reply:
Done in https://github.com/elastic/elasticsearch/pull/117917/commits/f170bdd912ed2212860201b05ad720fd9bad33fd